bears/go/GoTOMLBear.py

```diff
@@ -0,0 +1,23 @@
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.GoRequirement import GoRequirement
+
+
+@linter(executable='gotoml',
+        use_stdin=True,
+        output_format='corrected')
+class GoTOMLBear:
+    """
+    It is used to the know the type of every key used in a file.
+    """
+    LANGUAGES = {'Go'}
+    REQUIREMENTS = {
+               GoRequirement(package='github.com/BurntSushi/toml/cmd/tomlv',
```

Comment:
"package=" is not necessary.
See my comment from 3 hrs ago:
https://github.com/coala/coala-bears/pull/1404#discussion_r99460985